erpnext/schools/doctype/fees/fees.py

```diff
@@ -27,6 +78,64 @@ def get_fee_list(doctype, txt, filters, limit_start, limit_page_length=20, order
 			order by due_date asc limit {0} , {1}'''
 			.format(limit_start, limit_page_length), student, as_dict = True)
 
+
+@frappe.whitelist()
+def get_payment_entry(dt, dn, bank_account=None):
```

Comment:
Can't it be commonfied with `https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/doctype/payment_entry/payment_entry.py#L667`?